packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx

```diff
@@ -46,6 +46,12 @@ const CoreProperties: React.FC<CorePropertiesProps> = ({ market, reportingBarSho
               label="Event Expiration"
               time={market.endTimeFormatted}
               showLocal
+              hint={
+                <>
+                  <h4>Event expiration</h4>
+                  <p>This date time indicates when the settlement process begins.</p>
```

Comment:
add this string in a constant so we don't have multiple instances of it. most def it's going to change